cmd/agent/daemon/state/controller.go

```diff
@@ -128,31 +119,32 @@ func NewController(
 		ct:                         ct,
 		tracer:                     tracer,
 		signatureEngine:            signatureEngine,
-		enrichmentService:          enrichmentService,
 		kubeClient:                 kubeClient,
 		nodeName:                   os.Getenv("NODE_NAME"),
 		containerStatsScrapePoints: map[uint64]*containerStatsScrapePoint{},
 		mutedNamespaces:            map[string]struct{}{},
 		dnsCache:                   dnsCache,
 		podCache:                   podCache,
-		conntrackCache:             conntrackCache,
 		processTreeCollector:       processTreeCollector,
 		procHandler:                procHandler,
+		eventsEnrichmentService:    eventsEnrichmentService,
+		deletedContainersQueue:     make(chan uint64, 200),
```

Comment:
Should the size of this queue be configurable as we do for EventsBatchSize?

Reply:
This is more of a internal size, users should not need to change it. I increased it a bit to have 300 container deletes at the same time without any blocking.